lib/busted/middleware/busted_tracer.rb

```diff
@@ -0,0 +1,26 @@
+require "logger"
+
+class BustedTracer
```

Comment:
I looked at a few other middlewares, and they are using a different class name and file structure.

 - [`SecureHeaders::Middleware`](https://github.com/twitter/secureheaders/blob/7801462a4ea6c83479d7c38f6820e56177610745/lib/secure_headers/middleware.rb)
 - [`BetterErrors::Middleware`](https://github.com/charliesome/better_errors/blob/40cb4e6f6a4cdd0bc67a21a45bfadf9d325d1305/lib/better_errors/middleware.rb)
 - [`RequestStore::Middleware`](https://github.com/steveklabnik/request_store/blob/2b5e071fe93243b9dbca9df04dc3798b99be0c04/lib/request_store/middleware.rb)

I think we should adopt this pattern.

 - `Busted::Middleware`
 - `./lib/busted/middleware.rb`

Thoughts @HParker ?